### ISSUE TITLE
Add recommended action for auditd

### ISSUE THREAD

**Issue by vpetersson** [WoTTsecurity/api#658]
Logic: Check if the auditd package is installed

Severity: Medium

**Title**
Consider installing auditd for improved security logs

**Tl;dr**
\`auditd\` is part of the Linux Auditing System and is used in a lot of security minded organizations to ensure proper audit logs. While \`auditd\` by itself is very useful, it is particularly useful in conjunction with a central logging to ensure an attacker cannot clean up the logs.

**FAQ**
In most compliance environments, \`auditd\` (and the Linux Auditing System) plays an important role to ensure there are proper audit trails on the servers. Out-of-box, \`auditd\` provides good logging for system events, and it is possible to expand it further with custom rules.

You can see the result of \`auditd\` both in the system logs and using the \`aureport\` command. To learn more, see the \[man pages\](\https://linux.die.net/man/8/auditd).

Please note that if you are using \`auditd\`, it is strongly advised that you also set up remote logging using something like 'rsyslog' or 'syslog-ng`

**Code Snippet**

To install \`auditd\` on a Debian or Ubuntu system, run:

\`\`\`
$ sudo apt-get install auditd
\`\`\`
